#get_workspaces_on_map

string map

# flag: if need objectsInfo. False: objectsInfo will be empty. 
# when SRSSPatialInfo unavailable, corresponding items (e.g. x, y, z) will be set -1000 which is certainly invalid for srs. Or the whole will be set null
# default: true
bool ifGeometryInfo

---

##### TODO: Better to wrap everything in one msg file. Same for other similar srv.. 

string[] objects
string[] classesOfObjects
srs_msgs/SRSSpatialInfo[] objectsInfo
string[] houseHoldId
string[] readableNames

string[] json_properties   # other properties in Json, such as Fridge inRoom Kitchen etc. 